File: android/src/com/mygdx/game/ads/InterstitialAd.java
package com.mygdx.game.ads;

import android.widget.Toast;

import com.mygdx.game.AndroidLauncher;

public class InterstitialAd implements AdInterface {

    private AndroidLauncher context;

    public InterstitialAd(AndroidLauncher context){
        this.context = context;
    }

    @Override
    public void showAd() {

        context.runOnUiThread(new Runnable() {
            public void run() {
                Toast.makeText(context, "PERDISTE", Toast.LENGTH_LONG).show();
            }
        });
    }
}

File: android/src/com/mygdx/game/ads/InterstitialAd.kt
package com.mygdx.game.ads;

import com.mygdx.game.AndroidLauncher

class InterstitialAd(context : AndroidLauncher) : AdInterface {

    var firstInterstitialProvider: Provider<*>? = null

    init {
        firstInterstitialProvider = FbInterstitialProvider(context, null) //AdmobInterstitialProvider(context, null)
        firstInterstitialProvider?.load()
    }


    override fun showAd() {
        firstInterstitialProvider?.show()
    }


}
